circe.el

```diff
@@ -1949,7 +1956,12 @@ setting the variable through network options."
   (with-circe-server-buffer
     (let* ((target-name (irc-isupport--case-fold (circe-server-process)
                                                  target))
-           (chat-buffer (generate-new-buffer target))
+           (chat-buffer (generate-new-buffer
+                         (lui-format 'circe-format-buffer-name
+                                     `(target
```

Comment:
Please extract the argument to `generate-new-buffer` into a separate let binding, like `buffer-name` or something.

Reply:
Thanks, this has been done.